libs/datasets/timeseries.py

```diff
@@ -74,16 +74,16 @@ def latest_values(self, aggregation_level=None) -> pd.DataFrame:
 
         Return: DataFrame
         """
-        if not aggregation_level:
+        if aggregation_level is None:
             county = self.latest_values(aggregation_level=AggregationLevel.COUNTY)
             state = self.latest_values(aggregation_level=AggregationLevel.STATE)
             return pd.concat([county, state])
-
-        if aggregation_level == AggregationLevel.COUNTY:
+        elif aggregation_level == AggregationLevel.COUNTY:
```

Comment:
apparently the preferred way of comparing enums is with `is`.  i.e. `aggregation_level is AggregationLevel.COUNTY`

Reply:
Done